cardano-testnet/cardano-testnet.cabal

```diff
@@ -189,6 +189,8 @@ test-suite cardano-testnet-test
                         Cardano.Testnet.Test.Node.Shutdown
                         Cardano.Testnet.Test.SubmitApi.Babbage.Transaction
 
+                        Cardano.Testnet.Test.Utils
```

Comment:
That's not a test suite - it should go into the library.

Reply:
Solved in https://github.com/IntersectMBO/cardano-node/pull/5731/commits/6d6616b8171501f47fa1b9a4d60b2a6054c9ce92